openpype/hosts/blender/api/plugin.py

```diff
@@ -1345,6 +1039,358 @@ def remove(self, container: Dict) -> bool:
+            libpath (Path): Path of library to link.
+            container_name (str): Name of container to link.
+            container (OpenpypeContainer): Load into existing container.
+                Defaults to None.
+            override (bool, optional): Apply library override to linked
+                datablocks. Defaults to True.
+
+        Returns:
+            Tuple[List[bpy.types.ID], OpenpypeContainer]:
+                (Created scene container, Linked datablocks)
+        """
+        # Load collections from libpath library.
+        container, all_datablocks = self._load_library_datablocks(
+            libpath, container_name, container=container, do_override=override
+        )
+
+        container_collection = container.outliner_entity
+        if container_collection:
+            # If override_hierarchy_create method is not implemented for older
+            # Blender versions we need the following steps.
+            if not hasattr(container_collection, "override_hierarchy_create"):
+                link_to_collection(
+                    container_collection, bpy.context.scene.collection
+                )
+                container_collection = container_collection.override_create(
+                    remap_local_usages=True
+                )
+
+                for child in get_children_recursive(container_collection):
+                    child.override_create(remap_local_usages=True)
+
+                for obj in set(container_collection.all_objects):
+                    obj.override_create(remap_local_usages=True)
+
+                # force remap to fix modifers, constaints and drivers targets.
+                for obj in set(container_collection.all_objects):
+                    obj.override_library.reference.user_remap(obj.id_data)
+
+        return container, all_datablocks
+
+    def _append_blend(
+        self,
+        libpath: Path,
+        container_name: str,
+        container: OpenpypeContainer = None
+    ) -> Tuple[OpenpypeContainer, List[bpy.types.ID]]:
+        """Append blend process.
+
+        Args:
+            libpath (Path): Path of library to append.
+            container_name (str): Name of container to append.
+            container (OpenpypeContainer): Load into existing container.
+                Defaults to None.
+
+        Returns:
+            Tuple[List[bpy.types.ID], OpenpypeContainer]:
+                (Created scene container, Appended datablocks)
+        """
+        # Load collections from libpath library.
+        container, all_datablocks = self._load_library_datablocks(
+            libpath, container_name, container=container, link=False
+        )
+
+        # Link loaded collection to scene
+        container_collection = container.outliner_entity
+        if container_collection:
+            link_to_collection(
+                container_collection, bpy.context.scene.collection
+            )
+
+        return container, all_datablocks
+
+    def _instance_blend(
```

Comment:
Because we are refactoring, this one could be moved right after the `_link_blend`.